src/main/java/net/mcreator/element/types/Plant.java

```diff
@@ -53,6 +54,8 @@
 
 	public String tintType;
 	public boolean isItemTinted;
+	public NumberProcedure blockBounciness;

```

Comment:
We keep related parameters together, blockBounciness and fallDamageInduced are not related to tinting group, so move parameter location to more approriate location, or make new group by adding new line. above them